src/test/java/org/cactoos/io/FileAsInputTest.java

```diff
@@ -44,14 +46,16 @@
      */
     @Test
     public void readsFileContent() throws IOException {
+        final Charset utf = StandardCharsets.UTF_8;
```

Comment:
@g4s8 just use it inline. What's the point of a new variable? Remember, we're trying to keep our tests statement-free, as this article recommends: http://www.yegor256.com/2017/05/17/single-statement-unit-tests.html